sda-commons-server-prometheus/src/test/java/org/sdase/commons/server/prometheus/PrometheusBundleTest.java

```diff
@@ -199,6 +202,8 @@ void micrometerMetricsAvailableInPrometheus() {
     assertThat(sampleList.get(0).labelValues).contains("testTagValue");
 
     assertThat(sampleList.get(0).value).isEqualTo(2);
+
+    Metrics.globalRegistry.close();
```

Comment:
Do we need to do this in a `finally` block or `afterEach`?

Reply:
I decided to use `Metrics.globalRegistry.clear()` in `afterEach` see comments above. For some tests it might be necessary to use removeRegistry

Reply:
But if we have to clear it here in tests, consumers must do that as well in their tests, don't they?

Reply:
I thinks that depends on the test.

If the metrics tests are in different classes the shutdown behaviour should kick in and clearing should be unnecessary. If the same metric should be tested multiple times in the same class then registry needs to be cleared in between the tests.